packages/adapter-next/src/index.ts

```diff
@@ -26,16 +27,16 @@ export default defineSliceMachinePlugin<PluginOptions>({
 		hook("slice:update", sliceUpdate);
 		hook("slice:delete", sliceDelete);
 		hook("slice:read", sliceRead);
+		hook("slice:library:read", sliceLibraryRead);
```

Comment:
What do you think about treating "Slice Library" as its own object?

`slice-library:read`

`slice:library:read` hierarchically conveys that a library is a child of Slice, but it's not.

Reply:
LGTM